src/jupyter-aou/docker-compose.yaml

```diff
@@ -0,0 +1,50 @@
+version: '2.4'
+services:
+  app:
+    container_name: "application-server"
+    image: "gcr.io/deeplearning-platform-release/tf-gpu.2-11.py310"
```

Comment:
deep learning image is the AoU is using

Reply:
Is pytorch not available for AoU VMs?

Reply:
Where/how are `terra-docker` images used? https://github.com/DataBiosphere/terra-docker

Reply:
nvm, I see that this image is used as the base for all of their images https://github.com/DataBiosphere/terra-docker/blob/master/terra-jupyter-base/Dockerfile#L3